<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ExtensionEditor</class>
 <widget class="QMainWindow" name="ExtensionEditor">
  <property name="enabled">
   <bool>true</bool>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>900</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>900</width>
    <height>600</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="animated">
   <bool>true</bool>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="gridLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>881</width>
      <height>571</height>
     </rect>
    </property>
    <layout class="QGridLayout" name="gridLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetDefaultConstraint</enum>
     </property>
     <item row="0" column="0" rowspan="3" colspan="2">
      <widget class="QGroupBox" name="ManifestBox">
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Manifest</string>
       </property>
       <widget class="QLineEdit" name="Name">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>39</y>
          <width>191</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLabel" name="label_Name">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Name</string>
        </property>
       </widget>
       <widget class="QLabel" name="label_Version">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>20</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Version</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="Version">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>40</y>
          <width>81</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
       <widget class="QLabel" name="label_Publisher">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>20</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Publisher</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="Publisher">
        <property name="geometry">
         <rect>
          <x>300</x>
          <y>40</y>
          <width>151</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>10</pointsize>
         </font>
        </property>
       </widget>
       <widget class="QLabel" name="label_PublicKey">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>521</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Public key</string>
        </property>
       </widget>
       <widget class="QLineEdit" name="PublicKey">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>540</y>
          <width>561</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <family>Consolas</family>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="readOnly">
         <bool>false</bool>
        </property>
       </widget>
       <widget class="QLabel" name="label_Description">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>71</y>
          <width>91</width>
          <height>20</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Description</string>
        </property>
       </widget>
       <widget class="QPlainTextEdit" name="Description">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>90</y>
          <width>561</width>
          <height>41</height>
         </rect>
        </property>
       </widget>
       <widget class="QGroupBox" name="Icons">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>390</y>
          <width>561</width>
          <height>121</height>
         </rect>
        </property>
        <property name="title">
         <string>Icons</string>
        </property>
        <widget class="QFrame" name="x48">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>70</y>
           <width>281</width>
           <height>51</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLineEdit" name="icon3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>251</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_x48">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>2</y>
            <width>91</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>48x48px</string>
          </property>
         </widget>
         <widget class="QToolButton" name="iconBtn3">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>20</y>
            <width>25</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <italic>false</italic>
            <bold>true</bold>
            <underline>false</underline>
            <strikeout>false</strikeout>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="mouseTracking">
           <bool>false</bool>
          </property>
          <property name="inputMethodHints">
           <set>Qt::ImhNone</set>
          </property>
          <property name="text">
           <string>...</string>
          </property>
          <property name="checkable">
           <bool>false</bool>
          </property>
          <property name="checked">
           <bool>false</bool>
          </property>
          <property name="autoRepeat">
           <bool>false</bool>
          </property>
          <property name="autoExclusive">
           <bool>false</bool>
          </property>
         </widget>
        </widget>
        <widget class="QFrame" name="x32">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>20</y>
           <width>281</width>
           <height>51</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLineEdit" name="icon2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>251</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_x32">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>2</y>
            <width>91</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>32x32px</string>
          </property>
         </widget>
         <widget class="QToolButton" name="iconBtn2">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>20</y>
            <width>25</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </widget>
        <widget class="QFrame" name="x128">
         <property name="geometry">
          <rect>
           <x>280</x>
           <y>70</y>
           <width>281</width>
           <height>51</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLineEdit" name="icon4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>251</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_x128">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>3</y>
            <width>91</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>128x128px</string>
          </property>
         </widget>
         <widget class="QToolButton" name="iconBtn4">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>20</y>
            <width>25</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </widget>
        <widget class="QFrame" name="x16">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>20</y>
           <width>281</width>
           <height>51</height>
          </rect>
         </property>
         <property name="frameShape">
          <enum>QFrame::StyledPanel</enum>
         </property>
         <property name="frameShadow">
          <enum>QFrame::Raised</enum>
         </property>
         <widget class="QLineEdit" name="icon1">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>251</width>
            <height>20</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <family>Consolas</family>
           </font>
          </property>
          <property name="readOnly">
           <bool>true</bool>
          </property>
         </widget>
         <widget class="QLabel" name="label_x16">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>2</y>
            <width>91</width>
            <height>20</height>
           </rect>
          </property>
          <property name="text">
           <string>16x16px</string>
          </property>
         </widget>
         <widget class="QToolButton" name="iconBtn1">
          <property name="geometry">
           <rect>
            <x>250</x>
            <y>20</y>
            <width>25</width>
            <height>21</height>
           </rect>
          </property>
          <property name="font">
           <font>
            <weight>75</weight>
            <bold>true</bold>
           </font>
          </property>
          <property name="cursor">
           <cursorShape>PointingHandCursor</cursorShape>
          </property>
          <property name="text">
           <string>...</string>
          </property>
         </widget>
        </widget>
       </widget>
       <widget class="QListWidget" name="permissions">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>160</y>
          <width>191</width>
          <height>221</height>
         </rect>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::AllEditTriggers</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_Permissions">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>140</y>
          <width>131</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Permissions</string>
        </property>
       </widget>
       <widget class="QListWidget" name="content_scripts">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>160</y>
          <width>161</width>
          <height>221</height>
         </rect>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::AllEditTriggers</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_content_scripts">
        <property name="geometry">
         <rect>
          <x>210</x>
          <y>140</y>
          <width>181</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Content Scripts</string>
        </property>
       </widget>
       <widget class="QListWidget" name="web_accessible_resources">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>160</y>
          <width>191</width>
          <height>221</height>
         </rect>
        </property>
        <property name="cursor" stdset="0">
         <cursorShape>PointingHandCursor</cursorShape>
        </property>
        <property name="editTriggers">
         <set>QAbstractItemView::AllEditTriggers</set>
        </property>
       </widget>
       <widget class="QLabel" name="label_web_accessible_resources">
        <property name="geometry">
         <rect>
          <x>380</x>
          <y>140</y>
          <width>171</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Web Accessible Resources</string>
        </property>
       </widget>
       <widget class="QComboBox" name="default_locale">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>40</y>
          <width>111</width>
          <height>22</height>
         </rect>
        </property>
       </widget>
       <widget class="QLabel" name="label_default_locale">
        <property name="geometry">
         <rect>
          <x>460</x>
          <y>20</y>
          <width>111</width>
          <height>21</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Default Locale</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="2" column="2">
      <widget class="QFrame" name="commands">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>72</height>
        </size>
       </property>
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>72</height>
        </size>
       </property>
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <widget class="QPushButton" name="Build">
        <property name="geometry">
         <rect>
          <x>150</x>
          <y>40</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Build</string>
        </property>
       </widget>
       <widget class="QPushButton" name="Save">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>40</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Save</string>
        </property>
       </widget>
       <widget class="QPushButton" name="ThemeEditor">
        <property name="geometry">
         <rect>
          <x>0</x>
          <y>0</y>
          <width>141</width>
          <height>31</height>
         </rect>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
         </font>
        </property>
        <property name="text">
         <string>Theme Editor</string>
        </property>
       </widget>
      </widget>
     </item>
     <item row="0" column="2">
      <widget class="QGroupBox" name="BuildOptions">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>305</height>
        </size>
       </property>
       <property name="font">
        <font>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="title">
        <string>Build Options</string>
       </property>
       <widget class="QGroupBox" name="BrowserType">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>20</y>
          <width>271</width>
          <height>71</height>
         </rect>
        </property>
        <property name="title">
         <string>Browser Type</string>
        </property>
        <widget class="QRadioButton" name="OptionChromium">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>101</width>
           <height>17</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Chromium</string>
         </property>
        </widget>
        <widget class="QRadioButton" name="OptionFirefox">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>81</width>
           <height>17</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <pointsize>10</pointsize>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>Firefox</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>100</y>
          <width>271</width>
          <height>91</height>
         </rect>
        </property>
        <property name="title">
         <string>Minification</string>
        </property>
        <widget class="QCheckBox" name="minifyHTML">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>91</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Minify HTML</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="minifyCSS">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>81</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Minify CSS</string>
         </property>
        </widget>
        <widget class="QCheckBox" name="minifyJS">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>71</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>Minify JS</string>
         </property>
        </widget>
       </widget>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="geometry">
         <rect>
          <x>10</x>
          <y>200</y>
          <width>271</width>
          <height>91</height>
         </rect>
        </property>
        <property name="title">
         <string>Build CRX</string>
        </property>
        <widget class="QLineEdit" name="prv_key">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>60</y>
           <width>251</width>
           <height>20</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <family>Consolas</family>
          </font>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
        <widget class="QCheckBox" name="CRX">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>20</y>
           <width>51</width>
           <height>17</height>
          </rect>
         </property>
         <property name="text">
          <string>CRX</string>
         </property>
        </widget>
        <widget class="QToolButton" name="prv_key_button">
         <property name="geometry">
          <rect>
           <x>240</x>
           <y>60</y>
           <width>25</width>
           <height>21</height>
          </rect>
         </property>
         <property name="font">
          <font>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="cursor">
          <cursorShape>PointingHandCursor</cursorShape>
         </property>
         <property name="text">
          <string>...</string>
         </property>
        </widget>
        <widget class="QLabel" name="label">
         <property name="geometry">
          <rect>
           <x>10</x>
           <y>40</y>
           <width>251</width>
           <height>16</height>
          </rect>
         </property>
         <property name="text">
          <string>Private Key</string>
         </property>
        </widget>
       </widget>
      </widget>
     </item>
     <item row="1" column="2">
      <spacer name="verticalSpacer">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>20</width>
         <height>40</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>900</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Extension</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionBuild"/>
    <addaction name="separator"/>
    <addaction name="actionCopyPublicKey"/>
   </widget>
   <widget class="QMenu" name="menuModding">
    <property name="title">
     <string>Modding</string>
    </property>
    <addaction name="actionTheme_Editor"/>
   </widget>
   <addaction name="menuTest"/>
   <addaction name="menuModding"/>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionBuild">
   <property name="text">
    <string>Build</string>
   </property>
  </action>
  <action name="actionCopyPublicKey">
   <property name="text">
    <string>Copy Public Key</string>
   </property>
  </action>
  <action name="actionReveal">
   <property name="text">
    <string>Reveal in File Explorer</string>
   </property>
  </action>
  <action name="actionTheme_Editor">
   <property name="text">
    <string>Theme Editor</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
